rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -1810,6 +1810,141 @@ public void rskTxWaitingForSignature_uses_updateCollection_rskTxHash_before_rski
         assertEquals(0, provider.getReleaseTransactionSet().getEntries().size());
     }
 
+    @Test
+    public void rskTxWaitingForSignature_postRSKIP326_emitNewPegoutConfirmedEvent() throws IOException {
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP326)).thenReturn(true);
+
+        BridgeConstants spiedBridgeConstants = spy(BridgeRegTestConstants.getInstance());
+        doReturn(1).when(spiedBridgeConstants).getRsk2BtcMinimumAcceptableConfirmations();
+
+        BridgeStorageProvider provider = mock(BridgeStorageProvider.class);
+        BtcTransaction btcTx = mock(BtcTransaction.class);
+        Set<ReleaseTransactionSet.Entry> releaseTransactionSetEntries = new HashSet<>();
+        long rskBlockNumber = 1L;
```

Comment:
The rskBlockNumber Can Be More Realistic Using Figures like 1020L, 2345L

Reply:
I used 1 because that's what all the other tests have.